KsApi/combine/CombineTestObserver.swift

```diff
@@ -0,0 +1,16 @@
+import Combine
+import Foundation
+
+public final class CombineTestObserver<Value, Error: Swift.Error> {
```

Comment:
There's another ticket for fleshing this out. This is just the barest-bones stub to get the tests working.